sample/build.gradle

```diff
@@ -36,16 +36,16 @@ android {
 
 dependencies {
     implementation(
-        project(':survey-sdk')
+        project(':survey-sdk'),
+        "androidx.activity:activity-ktx:1.2.1",
+        "androidx.appcompat:appcompat:1.3.0",
+        "androidx.constraintlayout:constraintlayout:2.0.0-alpha3",
+        "androidx.core:core-ktx:1.3.0",
+        "androidx.lifecycle:lifecycle-runtime-ktx:2.4.0-alpha02",
+        'com.google.android.material:material:1.4.0'
```

Comment:
I think we should keep our sample as simple as possible. Do we need those dependencies?
- androidx.activity:activity-ktx
- androidx.lifecycle:lifecycle-runtime-ktx
- com.google.android.material:material

Reply:
@luongvo these are what I get from template. So I suggest we will make the decision at the last PR as this is only sample package and it will be changed in each PR.

Reply:
@doannimble I don't have any comment about the current dependencies but at least the `'com.google.android.material:material:1.4.0'` should be wrapped by double quote `"` instead 😃

Reply:
@hoangnguyen92dn Agree, to make the code consistency 👍